### ISSUE TITLE
Document Heros

### ISSUE THREAD

**Issue by nikolaswise** [Esri/calcite-web#299]
from UC user testing
### "Full-screen hero shots conveyed little information and inhibited scrolling"

This is also a sort of known problem. The current sub-nav-as-hero pattern forces a developer to work for their splashy screen, and the suggested patterns currently restrain the amount of padding around content in a hero to a modest amount (up to 4 or 5 baseline measures). Moving to a hard setting of 65vh as max will cause some for real serious problems. Better to make clear suggestions about the proper use of heros. 

In short:
#### BAD

http://desktop.arcgis.com/en/
#### GOOD / BETTER

http://storymaps.arcgis.com/en/
